api.md

```diff
@@ -2013,10 +2093,10 @@ removes the existing reorder policy for the `conditions` table if it exists.
 ---
 
 
-## alter_job_schedule() :community_function: [](alter_job_schedule)
+## alter_job() :community_function: [](alter_job)
 
 Policy jobs are scheduled to run periodically via a job run in a background
-worker. You can change the schedule using `alter_job_schedule`. To alter an
+worker. You can change the schedule using `alter_job`. To alter an
 existing job, you must refer to it by `job_id`. The `job_id` which implements a
 given policy and its current schedule can be found in views in the
 `timescaledb_information` schema corresponding to different types of policies or
```

Comment:
we don't seem to have policy specific views anymore. Are you planning to add one which shows all the polices? This part needs an update.